keygen

```diff
@@ -25,40 +46,51 @@ if [ "$OPENSSL_VERSION_MAJOR" -lt 1 ]; then
   exit 1
 fi
 
+if [ -z "$CERT_CN" ]; then
+    echo "Common name (CERT_CN) not set."
+    echo -n "Enter a certificate common name [docker.mender.io]:"
+    IFS='' read -r CERT_CN
+    if [ -z "$CERT_CN" ]; then
+        CERT_CN="docker.mender.io"
+    fi
+fi
+
+if [ -z "$CERT_SAN" ]; then
+    CERT_SAN="DNS:${CERT_CN},DNS:*.${CERT_CN}"
+    echo "Subject alternative name (CERT_SAN) not set using default:"
```

Comment:
just since it confused me for a second 🙈:
```suggestion
    echo "Subject alternative name (CERT_SAN) not set, using default:"
```

Reply:
That is confusing indeed. Good catch :+1: